ui/imports/Themes/DarkTheme.qml

```diff
@@ -2,27 +2,27 @@ import QtQuick 2.13
 import "."
 
 Theme {
-    property color white: "#FFFFFF"
+    property color white: "#141414"
     property color white2: "#FCFCFC"
-    property color black: "#000000"
-    property color grey: "#141414"
-    property color lightBlue: "#ECEFFC"
+    property color black: "#FFFFFF"
```

Comment:
Hm.. this doesn't look right to me `black` is `#FFFFFF`?

We're in dark theme here, so I assume it's just because the variable is named `black`. If that's the case we might want to consider renaming those colors to things like `fgColor`, `bgColor`, `primaryBgColor`, `secondaryBgColor` etc..

Reply:
I agree with @PascalPrecht 

In DarkTheme.qml and LightTheme.qml this is handled similar to what he described:

```qml
    property color background: "#141414"
    property color border: "#252528"
    property color textColor: white
    property color currentUserTextColor: white
    property color secondaryBackground: "#23252F"
    property color inputBackground: secondaryBackground
    property color inputColor: darkGrey
    property color modalBackground: background
```

Notice how the property names describe the usage instead of colors

Reply:
I agree with both, because that's how I did it haha.

I understand why you did it @emizzle , it's how the design shows it, but it's **super** counter-intuitive. So it's better to keep using the sub properties and keep the `white` as  `white` and just change those that make sense, eg: `grey` can be darker in dark mode, but it's still a `grey`